pathz/authorization/authorization.proto

```diff
@@ -37,11 +37,11 @@
 //
 syntax = "proto3";
 
-package gnsi.authorization;
```

Comment:
you don't need to namespace this the authorization proto can just reside in gnsi.pathz

Reply:
done. I just removed this package line entirely.
and the upper directory pathz.proto imports this proto... into pathz.